webtau-http/src/test/java/com/twosigma/webtau/http/HttpJavaTest.java

```diff
@@ -238,6 +239,25 @@ public void redirectPut() {
         });
     }
 
+    @Test
```

Comment:
is there anything queryParam-related property we can test for to validate it honored the parameter as expected?

Reply:
I'll change this to the params version because the test service is set up to respond with success to the params resource only if the appropriate query params are specified.